lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,186 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
+        'use strict';
+        describe('edx.commerce.ReceiptView', function() {
+            var data, courseResponseData, providerResponseData = null;
+
+            var createReceiptView = function() {
+                return new ReceiptView({el: $('#receipt-container')});
+            };
+
+            var loadReceiptFixture = function() {
+                var receiptFixture = readFixtures('templates/commerce/receipt.underscore');
+                var providerFixture = readFixtures('templates/commerce/provider.underscore');
+                appendSetFixtures(
+                    '<script id=\"receipt-tpl\" type=\"text/template\" >' + receiptFixture + '</script>'
+                );
+                appendSetFixtures(
+                    '<script id=\"provider-tpl\" type=\"text/template\" >' + providerFixture + '</script>'
+                );
+            };
+
+            var loadNonVerifiedReceiptFixture = function() {
+                var receiptFixture = readFixtures('js/fixtures/commerce/non_verified_receipt.html');
+                appendSetFixtures(receiptFixture);
+            };
+
+            var loadVerifiedReceiptFixture = function() {
+                var receiptFixture = readFixtures('js/fixtures/commerce/verified_receipt.html');
+                appendSetFixtures(receiptFixture);
+            };
+
+            var mockRequests = function(requests, method, apiUrl, responseData) {
+                 AjaxHelpers.expectRequest(requests, method, apiUrl);
+                 AjaxHelpers.respondWithJson(requests, responseData);
+            };
+
+            var mockRender = function() {
```

Comment:
There should only be 1 var declaration at the top of the function, comma separating the variable names.